glossary.md

```diff
@@ -479,6 +482,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | timeseries analysis                    | phân tích dữ liệu chuỗi thời gian         |                                              |
 | timestep                               | bước thời gian                            | [https://git.io/JvojQ](https://git.io/JvojQ) |
 | token                                  | token                                     |                                              |
+| total variation | biến thiên toàn phần |  |
```

Comment:
Total variation là bài toán tính tổng sự thay đổi giữa điểm lân cận theo Lp norm. Thường là L1 hoặc L2. TV được tính dựa trên gradient energy. 

Ngoài từ "biến thiên toàn phần", còn lựa chọn nào khác không nhỉ?